dashboard/assets/admin/dashboard/styles/custom.css

```diff
@@ -435,3 +435,12 @@ th.required::before {
 .form-row a[rel="external"]:after {
     background-position: right 3px;
 }
+
+div.inline-group div.inline-related fieldset.module table tr.toggle-collapsed-inlines td {
+    cursor: pointer;
+    height: 10px;
+    background-color: #f8f8f8;
+    border-bottom: 1px solid #eee;
+    text-align: center;
```

Comment:
might be better to align to the left. it tends to be more noticeable by most of the people.